contracts/interfaces/0.8.x/IGenArt721CoreContractV3.sol

```diff
@@ -23,6 +23,7 @@ interface IGenArt721CoreContractV3 {
     function coreType() external view returns (string memory);
 
     // owner (pre-V3 was named admin) of contract
+    // this is expected to be an Admin ACL contract for V3
```

Comment:
We may want to expose a readable field for the ACL checker of type `IAdminACLV0` in the interface itself for introspect-ability reasons–WDYT?

Reply:
yes good call. will update!
Side note, I definitely had a blind spot here with respect to "internalType" return types in ABIs post solidity v5.<something> (ref: https://ethereum.stackexchange.com/questions/76953/what-is-the-purpose-of-internaltype-now-generated-by-the-solidity-compiler-in)

Reply:
(updated)

Reply:
🙌 🙌 🙌 

>  Side note, I definitely had a blind spot here with respect to "internalType" return types in ABIs post-solidity v5.something (ref: https://ethereum.stackexchange.com/questions/76953/what-is-the-purpose-of-internaltype-now-generated-by-the-solidity-compiler-in)

Ah interesting–that was a blind spot for me as well. 😅